content/blog/2020/08/2020-08-04-cdf-graduation.adoc

```diff
@@ -0,0 +1,180 @@
+
+Another important CDF membership benefit is community outreach and marketing. 
+CDF helped us to establish connections with other CI/CD projects and end user companies.
+Through CDF we have access to the link:https://jenkins.devstats.cd.foundation/[DevStats service].  It provides community contribution statistics and helps us track trends and discover areas for improvement.
+On the marketing side, the foundation organizes webinars, podcasts and newsletters.
+Jenkins is regularly represented there.
+CDF also runs the meetup.com professional account which is used by local Jenkins communities for
+link:/projects/jam/[CI/CD and Jenkins Area Meetups].
+Last but not least, the Jenkins community is also represented at virtual conferences where CDF has a booth.
+All of that helps to grow Jenkins visibility and to highlight new features and initiatives in the project.
+
+== Why did we graduate?
+
+image::/images/logos/graduation/graduation.png[alt="Jenkins Graduation Logo", float=right, width="224px"]
+
+The Jenkins project has a long history of open governance which is a key part of today's project success.
+Starting from **2011**, the project has introduced link:/project/governance-meeting/[the governance meeting] which are open to anyone.
+Most of the discussions and decision making happen publicly in the link:/mailing-lists/[mailing lists].
+**In 2015** we introduced teams, sub-projects and officer roles.
+**In 2017** we introduced the link:https://github.com/jenkinsci/jep[Jenkins Enhancement Proposal] process which helped us to make the key architecture and governance decisions more open and transparent to the community and the Jenkins users.
+**In 2018** we introduced link:/sigs[special interest groups] that focus on community needs.
+**In 2019** we have link:/blog/2019/12/16/board-election-results/[expanded the Jenkins governance board]
+so that it got more bandwidth to facilitate initiatives in the project.
+
+Since the Jenkins project inception 15 years ago, it has been steadily growing.
+Now it has millions of users and thousands of contributors.
+In 2019 it has seen **5,433 contributors** from 111 countries and 272 companies,
+67 core and 2,654 plugin releases,
+45,484 commits, 7,000+ pull requests.
+In 2020 Q2 the project has seen **21% growth** in pull requests numbers compared to 2019 Q2, bots excluded.
+
+
+One may say that the Jenkins project already has everything needed to succeed.
+It is a result of continuous work by many community members,
+and this work will never end as long as the project remains active.
+Like in any other industry, the CI/CD ecosystem changes every day and sets new expectations from the automation tools in this domain.
+Same as the tools, the open source communities need to evolve so that they can address expectations, and onboard more users and contributors.
```

Comment:
```suggestion
Just as the tools evolve, open source communities need to evolve so that they can address expectations, and onboard more users and contributors.
```